<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CXLazerDirectionDialog</class>
 <widget class="QDialog" name="CXLazerDirectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Переехать</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="CXTouchButton" name="mAbsoluteButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Абсолютное</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CXTouchButton" name="mRelativeButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Относительное</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="mYEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mXEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="maximumSize">
      <size>
       <width>332</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="5" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2">
       <widget class="CXTouchButton" name="mButtonDel">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Del</string>
        </property>
        <property name="shortcut">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="CXTouchButton" name="mButtonEnter">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Enter</string>
        </property>
        <property name="shortcut">
         <string>Enter, Return</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="CXTouchButton" name="mButtonBackspace">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>←</string>
        </property>
        <property name="shortcut">
         <string>Backspace</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="CXTouchButton" name="mButtonMinus">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="shortcut">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="CXTouchButton" name="mButtonPlus">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>+</string>
        </property>
        <property name="shortcut">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="CXTouchButton" name="mButton7">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>7</string>
        </property>
        <property name="shortcut">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CXTouchButton" name="mButton8">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>8</string>
        </property>
        <property name="shortcut">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="CXTouchButton" name="mButton9">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>9</string>
        </property>
        <property name="shortcut">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="CXTouchButton" name="mButton4">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>4</string>
        </property>
        <property name="shortcut">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CXTouchButton" name="mButton5">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>5</string>
        </property>
        <property name="shortcut">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="CXTouchButton" name="mButton6">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>6</string>
        </property>
        <property name="shortcut">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="CXTouchButton" name="mButton1">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="shortcut">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="CXTouchButton" name="mButton2">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="shortcut">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="CXTouchButton" name="mButton3">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>3</string>
        </property>
        <property name="shortcut">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="CXTouchButton" name="mButton0">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="shortcut">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="CXTouchButton" name="mCancelButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CXTouchButton</class>
   <extends>QPushButton</extends>
   <header>cxtouchbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mXEdit</tabstop>
  <tabstop>mYEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
